src/soc/ibm/power9/pstates.c

```diff
@@ -1,15 +1,23 @@
 /* SPDX-License-Identifier: GPL-2.0-only */
 
 #include "homer.h"
+#include "wof.h"
+#include <commonlib/region.h>
 #include <cpu/power/mvpd.h>
 #include <assert.h>
+#include <endian.h>
 #include <lib.h>
 #include <string.h>		// memcpy
 
 #define IDDQ_MEASUREMENTS    6
 #define MAX_UT_PSTATES       64     // Oversized
 #define FREQ_STEP_KHZ        16666
 
+// TODO: move this and similar defines in tor.c into some header
+#define ACTIVE_QUADS 6
```

Comment:
There is now MAX_QUADS_PER_CHIP in homer.h, change it where needed and I'll merge it